src/openrct2/world/Banner.h

```diff
@@ -10,7 +10,15 @@
 #pragma once
 
 #include "../common.h"
-#include "TileElement.h"
+#include "../ride/RideTypes.h"
+#include "Location.hpp"
+
+#include <string>
+
+struct TileElement;
+struct WallElement;
+
+typedef uint16_t BannerIndex;
```

Comment:
Prefer `using`
```suggestion
using BannerIndex = uint16_t;
```

Reply:
Our code uses `typedef` almost throughout.

Reply:
Well, one could argue that you just moved this from another file and changed that type and that one was with `using` :P

Reply:
> Our code uses typedef almost throughout.

This is mostly due to our codebase originally being in C.

Reply:
I don't see the point, but well...

Reply:
`using` is the C++ way of doing `typedef` since C++11. Technically they behave exactly like one another, but in terms of readability, I think `using` is friendlier to new programmers, due to the `X = Y` construction.

It's a point made by [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rt-using) and some of the examples show that it really adds value in some cases

Reply:
Can you open a PR to replace the rest, then? I prefer to be consistent.

Reply:
Will do!